### ISSUE TITLE
cli + matching-brackets: Exercise files are not downloaded

### ISSUE THREAD

**Issue by nkanderson** [exercism/rust#853]
Running `exercism download --exercise=matching-brackets --track=rust` on my computer does not seem to download any of the exercise files, like the `src` and `test` directories, `README.md`, etc. I've been able to download other exercises successfully.

The `matching-brackets` directory itself is created, as is the `.exercism` directory within it. I don't get any error message, just the typical `Downloaded to
<my exercism rust track directory>/matching-brackets`.

I initially encountered the issue with the exercism cli version 3.0.11. Tried upgrading and hit the same thing with 3.0.12. My OS info as reported by `exercism troubleshoot` is the following:
```
Operating System
----------------
OS:           darwin
Architecture: amd64
```

I'm not sure if this issue belongs on the project for the exercism CLI - it seems specific to this one exercise, so I thought I'd start here.

**Comment by coriolinus** [comment]
Hmm. I couldn't replicate this locally:

```sh
coriolinus@brute /mnt/c/Users/coriolinus/Documents/Projects/exercism
$ ./exercism version
Mon Aug 12 19:57:46 DST 2019
exercism version 3.0.12
coriolinus@brute /mnt/c/Users/coriolinus/Documents/Projects/exercism
$ uname -a
Mon Aug 12 19:57:49 DST 2019
Linux brute 4.4.0-17763-Microsoft #379-Microsoft Wed Mar 06 19:16:00 PST 2019 x86_64 x86_64 x86_64 GNU/Linux
$ rm -rf matching-brackets/
Mon Aug 12 19:58:57 DST 2019
coriolinus@brute /mnt/c/Users/coriolinus/Documents/Projects/exercism/rust
$ ../exercism download --exercise=matching-brackets --track=rust
Mon Aug 12 19:59:00 DST 2019

Downloaded to
/mnt/c/Users/coriolinus/Documents/Projects/exercism/rust/matching-brackets
coriolinus@brute /mnt/c/Users/coriolinus/Documents/Projects/exercism/rust
$ tree matching-brackets/
Mon Aug 12 19:59:05 DST 2019
matching-brackets/
├── Cargo.toml
├── README.md
├── src
│   └── lib.rs
└── tests
    └── matching-brackets.rs

2 directories, 4 files
```

There is one possibility I can think of which might be causing this behavior for you: in https://github.com/exercism/rust/pull/819, the exercise was renamed to `bracket-push`. It should have retained backwards compatibility, though.

It may be worth trying `exercism download --exercise=bracket-push --track=rust`. If that also fails, then I would suggest moving this issue to https://github.com/exercism/configlet.

**Comment by nkanderson** [comment]
Oh, that renaming is interesting, and definitely seems like it could be related. My initial setup of exercism is fairly old (first installed Oct 2017), and I've downloaded all the Rust exercises over time. But since there are fairly regular updates, I usually download again just before working on one.

And as it turns out, I already have a `bracket-push` exercise downloaded. I tried moving the `bracket-push` directory out of my exercism directory, and I get the same result as above when attempting to download `matching-brackets`.

I may try to replicate this on another machine, and I'll take a look at that configlet project as well.

**Comment by coriolinus** [comment]
Don't worry about configlet; that was a mistake on my part. I meant to link you to https://github.com/exercism/cli.

**Comment by nkanderson** [comment]
Ok, thanks. I've confirmed this is an issue on another system, with a fresh install of exercism, so I'll create an issue on the cli project.

**Comment by nkanderson** [comment]
Closed by database fix in exercism/exercism#4980
